CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to Gitpod Samples
+
+Thank you for showing your interest in contributing to Gitpod Samples. We value community contributions 💜 
+
+We celebrate that every person brings a unique perspective and experience to the community. 
+
+Before reporting any issues or pull requests, please read through this document to ensure we have all the information to respond effectively to your contribution.
```

Comment:
I would change this to simply: 

Before reporting any issues or raising pull requests, please read through this document 🙌